cita-executor/core/src/executive.rs

```diff
@@ -326,11 +326,16 @@ impl<'a, B: 'a + StateBackend> Executive<'a, B> {
 
         let schedule = Schedule::new_v1();
 
+        let gas_used_by_data_length = match t.version {
+            0...2 => 0,
+            _ => t.data.len() * schedule.create_data_gas,
```

Comment:
This value is `gas_used_by_data_length`, can't represent its actual meaning. Pls change the value name.